solr/core/src/java/org/apache/solr/search/SolrIndexSearcher.java

```diff
@@ -1921,8 +1921,11 @@ public ScoreMode scoreMode() {
       final TopDocs topDocs;
       final ScoreMode scoreModeUsed;
       if (!MultiThreadedSearcher.allowMT(pf.postFilter, cmd, query)) {
-        if (log.isDebugEnabled()) {
-          log.debug("skipping collector manager");
+        if (cmd.getMultiThreaded() != null) {
+          // report multi-threading (non-)use only if multi-threading was explicitly (not) requested
+          log.info("SINGLE THREADED search, skipping collector manager");
+        } else {
+          log.debug("SINGLE THREADED search, skipping collector manager");
```

Comment:
variation of @gus-asf's https://github.com/apache/solr/pull/2595/files#diff-99978700f1c69d6a5f6c2190f89c98cfe20c441161db5a183ec002e15cb1be28R1924 change.